src/file/AttributesFS.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2013-2015, German Neuroinformatics Node (G-Node)
+//
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted under the terms of the BSD License. See
+// LICENSE file in the root of the Project.
+
+#include "nix/file/AttributesFS.hpp"
+
+
+using namespace nix::file;
+using namespace boost::filesystem;
```

Comment:
Maybe namespace bfs = boost::filesystem. Shorter and we avoid possible conflicts like path, file, whatnot.